tssv/tssv.py

```diff
@@ -397,3 +417,4 @@ def tssv(
         write_files(tables, files)
 
     make_report(tables, report_handle)
+    make_json(tables, json_handle)
```

Comment:
Would it not be better to make the output format a parameter (e.g., text, json or none)?